src/congestion/BufferOccupancy.cc

```diff
@@ -71,19 +69,9 @@ void BufferOccupancy::initCredits(u32 _vcIdx, u32 _credits) {
 
   assert(port < numPorts_);
   assert(vc < numVcs_);
-  assert(_credits > 0);
+  assert(_credits > 0 || style() == CongestionSensor::Style::kAbsolute);
```

Comment:
slight preference for 

if (style() != kAbsolute) {
  assert(_credits > 0);
}

or should it be "if style() == kNormalize"?

Reply:
Fixed